jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/ColumnInfoDetails.kt

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+
+import com.intellij.util.ui.ColumnInfo
+import software.amazon.awssdk.services.cloudwatchlogs.model.ResultField
+import software.aws.toolkits.jetbrains.services.cloudwatch.logs.editor.LogStreamsStreamColumnRenderer
+import javax.swing.table.TableCellRenderer
+
+class ColumnInfoDetails(private val fieldName: String) : ColumnInfo<List<ResultField>, String>(fieldName) {
+    private val renderer = LogStreamsStreamColumnRenderer()
+    override fun valueOf(item: List<ResultField>?): String? {
```

Comment:
Could we just make this a map instead?

Reply:
The GetQueryResults call returns a [list of list of results fields](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/logs/model/GetQueryResultsResult.html#getResults--). A map is possible but would require some degree of preprocessing.